### PR TITLE
Add warning when inputting JSON in multi-opt description

### LINKED ISSUE #2138: Add warning when user pastes a multi-opt config inside multi-opt description
image
image

should be able to attempt to parse json of the description. if it is valid, then show the warning

### PR DESCRIPTION
**Pull request by clevernt**
## Describe your changes

Adds a warning when a user pastes a multi-opt config in a multi-opt description and tells them how to do it correctly.

## Issue or discord link

- Resolves #2138 

## Testing/validation

image

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [x] For front-end changes, I have updated the corresponding English translations.
- [x] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

File: libs/gi/ui/src/components/JsonDescWarning.tsx
Comment (frzyc) [inline libs/gi/ui/src/components/JsonDescWarning.tsx:1]: Since this is only used in `CustomMultiTargetCard`, I would advice moving this next to that file.

**Review by frzyc — CHANGES_REQUESTED** [review]

File: libs/gi/ui/src/components/JsonDescWarning.tsx
Comment (clevernt) [reply libs/gi/ui/src/components/JsonDescWarning.tsx:1]: I might be dumb but idk what you mean by that

Comment (frzyc) [reply libs/gi/ui/src/components/JsonDescWarning.tsx:1]: Move this file to `libs/gi/page-team/src/CharacterDisplay/CustomMultiTarget/JsonDescWarning.tsx`

**Review by frzyc — APPROVED** [review]
LGTM, thanks for your contribution!

**Pull request merged**